arrow-fx/src/main/kotlin/arrow/fx/Schedule.kt

```diff
@@ -708,10 +714,13 @@ sealed class Schedule<F, Input, Output> : ScheduleOf<F, Input, Output> {
 
     /**
      * Create a schedule that never retries.
```

Comment:
We should make clear here that it'll never return from `retry` or `repeat` until canceled.

This is useful in similar situations as `never` where you need to explicitly suspend until canceled or interrupted. You could compose it at the end of a `Schedule` using `andThen` to suspend after a `Schedule` finishes.

Reply:
added a note "Note that this will hang a program if used as a repeat/retry schedule unless cancelled."